### PR TITLE
feat: Product Page

### PR DESCRIPTION
**Pull request by arpit01923**
- Product Page

1. Product page tells about products presented in e-com and having three components Navigation, Main and Footer.
2. Navigation component is used to navigate logo, search, profile, wishlist and cart page, and for login or signup.
3. Main component is used for displaying header, drawer and products present in e-com.
4. Footer component is used for giving some extra information.

- What files to check :

1. src/pages/product/Product.jsx
2. src/pages/product/main/Main.jsx

Preview:

https://user-images.githubusercontent.com/86866251/159312392-a51fcf4c-f074-472c-8c75-2d4b0bbf158d.mp4

### CODE REVIEW

**Review by maitrakhatri — APPROVED** [review]
Code looks clean and is very easy to read

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-import {Home} from "./pages/home/Home"
+import { Product } from "./pages/product/Product";
+// import {Home} from "./pages/home/Home"
+
 
 function App() {
   return (
     <div className="App">
-      <Home />
+      {/* <Home /> */}
```
Comment (ark2002) [inline src/App.js]: should remove comments before commiting

File: src/components/navbar/Navbar.css [hunk 2]
```diff
@@ -1,38 +1,44 @@
-.logo {
-  width: 4rem;
-  height: 4rem;
-  margin: 0 1rem;
-}
-
-.nav-search {
-  padding: 0.2rem;
-}
+@import "../../App.css";
```
Comment (ark2002) [inline src/components/navbar/Navbar.css]: u dont need to import app.css explicitly in each component file it gets passed down through the hierarchy anyway

File: src/pages/product/Product.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
+@import "../../App.css";
```
Comment (ark2002) [inline src/pages/product/Product.css]: same here

File: src/pages/product/Product.jsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import "./Product.css";
+import { Navbar } from "../../components/navbar/Navbar";
+import { Footer } from "../../components/footer/Footer";
```
Comment (ark2002) [inline src/pages/product/Product.jsx]: can use barreling here for importing

File: src/pages/product/main/Dump.jsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  "https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTF8fHByb2R1Y3R8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60",
+  "https://images.unsplash.com/photo-1560343090-f0409e92791a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTB8fHByb2R1Y3R8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60",
+  "https://images.unsplash.com/photo-1546868871-7041f2a55e12?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTd8fHByb2R1Y3R8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60",
+  "https://images.unsplash.com/photo-1583394838336-acd977736f90?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTh8fHByb2R1Y3R8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60",
+  "https://images.unsplash.com/photo-1556228578-0d85b1a4d571?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MjN8fHByb2R1Y3R8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60",
+  "https://images.unsplash.com/photo-1491637639811-60e2756cc1c7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MjJ8fHByb2R1Y3R8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60",
+  "https://images.unsplash.com/photo-1585565804112-f201f68c48b4?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80"
+];
+
```
Comment (ark2002) [inline src/pages/product/main/Dump.jsx:15]: i am not getting how are you using the backend here. We are supposed to do an api call to get the products from the backend

**Review by ark2002 — APPROVED** [review]
Look for the suggestions

File: src/pages/product/Product.jsx (hunk 4 above)
Comment (arpit01923) [reply src/pages/product/Product.jsx]: yep

File: src/pages/product/main/Dump.jsx (hunk 5 above)
Comment (arpit01923) [reply src/pages/product/main/Dump.jsx:15]: it's just a dummy text, in future I should use backend

**Pull request merged**